jujugui/static/gui/src/app/user/user.js

```diff
@@ -160,6 +148,15 @@ const User = class User {
       });
     }
   }
+
+  clearMacaroons() {
+    this.localStorage.clear();
+  }
+
+};
+
+const cookieName = service => {
```

Comment:
This should be moved up in the file as the above methods are using something defined after their use